src/main/java/org/testeditor/fixture/host/screen/TerminalScreen.java

```diff
@@ -0,0 +1,277 @@
+        ArrayList<Field> fieldsPerline = new ArrayList<Field>();
+        // separate Fields "(c0=c1) 33 34 35" as String
+        int fieldCount = 0;
+        ArrayList<String> row = separateFields(rowAsHex);
+        for (String rawField : row) {
+            fieldsPerline.add(parseField(row, rawField, rowCount, ++fieldCount, rowAsHex));
+        }
+        rowCount = 0;
+        return fieldsPerline;
+    }
+
+    /**
+     * Parses the String reprentation of a Field in the format: "SF(c0=c1) 33 34
+     * 35 SF(c0=c1) 3d 36 37 38"
+     * 
+     * @param row
+     *            The complete List of all rawField String representations in
+     *            one row.
+     * @param rawField
+     *            String reprentation of a {@link Field} in the format: (c0=c1)
+     *            33 34 35
+     * @param rowCount
+     *            The row number on whole screen.
+     * @param fieldCount
+     *            The number of a field in one line.
+     * @param completeRowAsHex
+     *            Raw String of the whole line in the format: "data: SF(c0=e9)
+     *            7a 2f 4f 53 SF(c0=c0) 20 5a 31 38 20 4c 65 76 65 6c SF(c0=d1)
+     *            20 30 36 30 39"
+     * @return {@link Field}
+     */
+    private Field parseField(ArrayList<String> row, String rawField, int rowCount, int fieldCount,
+            String completeRowAsHex) {
+        // like separated "c0=c1", "33", "34", "35"
+        String[] splittedFieldAttributes = rawField.split(" ");
+        Field field = new Field();
+        field.setFieldAttributes(row, fieldCount);
+        field.setColumnNumber(rowCount);
+        field.setFieldCountPerLine(fieldCount);
+        field.setSplittedFieldAttributes(splittedFieldAttributes);
+        field.setControlCharacters(splittedFieldAttributes);
+        field.setCharacters(splittedFieldAttributes);
+        field.setStringRepresentation(field.getCharactersAsHex());
+        field.setControlCharacter(completeRowAsHex, fieldCount);
+        field.setHidden(field.isAttribute(field.getControlCharacter(), Field.DISPLAYABLE));
+        field.setNumeric(field.isAttribute(field.getControlCharacter(), Field.NUMERIC));
+        field.setProtected(field.isAttribute(field.getControlCharacter(), Field.PROTECTED));
+        return field;
+    }
+
+    /**
+     * 
+     * @param rowAsHex
+     *            Raw String of the whole line in the format: "data: SF(c0=e9)
+     *            7a 2f 4f 53 SF(c0=c0) 20 5a 31 38 20 4c 65 76 65 6c SF(c0=d1)
+     *            20 30 36 30 39"
+     * @return The complete List of all rawField String representations in one
+     *         row. String reprentation of a {@link Field} in the format:
+     *         "(c0=c1) 33 34 35", "(c0=c2) 36 37 38", ...
+     */
+    private ArrayList<String> separateFields(String rowAsHex) {
+        ArrayList<String> fieldValues = new ArrayList<String>();
+        String dataReplaced = rowAsHex.replace("data: ", "");
```

Comment:
'data: ' should be removed only at the start of the string! replace removes all occurances!